gcloud/language/sentiment.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Definition for Google Cloud Natural Language API sentiment.
+
+Sentiment is the response to an ``analyzeSentiment`` request.
+"""
+
+
+class Sentiment(object):
+    """A Google Cloud Natural Language API sentiment object.
+
+    .. _Sentiment message: https://cloud.google.com/natural-language/\
+                           reference/rest/v1beta1/Sentiment
+
+    See `Sentiment message`_.
+
+    :type polarity: float
+    :param polarity: Polarity of the sentiment in the ``[-1.0, 1.0]`` range.
+                     Larger numbers represent more positive sentiments.
```

Comment:
Logically, `0` could be a value, meaning "neutral".  But maybe that would show up via a `magnitude` approaching `0`?  If it can only be `1` or `-1`, then maybe we should be exposing a bool-like enum instead.